Sources/Refresher/Refresher.swift

```diff
@@ -100,13 +100,13 @@ public struct RefreshableScrollView<Content: View, RefreshView: View>: View {
     var refreshView: (Binding<RefresherState>) -> RefreshView
     
     @State private var headerInset: CGFloat = 1000000 // Somewhere far off screen
-    @State var state = RefresherState()
+    @State var state: RefresherState
     @State var distance: CGFloat = 0
     @State var rawDistance: CGFloat = 0
     @State var renderLock = false
-    private var style: Style
-    private var config: Config
-    
+    private let style: Style
```

Comment:
I'm changing the definition to 'let' because it seems there are no cases where the style is modified later."